File: 2026-04-April-GFG-POTD/April 08 - Segregate 0s and 1s/Solution.py
class Solution:
    def segregate0and1(self, arr):
        countZero = 0

        # Count zeros
        for num in arr:
            if num == 0:
                countZero += 1

        # Fill array
        for i in range(len(arr)):
            if i < countZero:
                arr[i] = 0
            else:
                arr[i] = 1

File: 2026-04-April-GFG-POTD/April 08 - Segregate 0s and 1s/Solution.cpp
class Solution {
public:
    void segregate0and1(vector<int>& arr) {
        int countZero = 0;

        // Count zeros
        for (int num : arr) {
            if (num == 0) countZero++;
        }

        // Fill array
        for (int i = 0; i < arr.size(); i++) {
            if (i < countZero) arr[i] = 0;
            else arr[i] = 1;
        }
    }
};
